### ISSUE TITLE
create a better list of materials

### ISSUE THREAD

**Issue by jintonic** [jintonic/shine#29]
The current list of materials are too long to navigate. Please split it to two. One for basic elements. the other for compounds. The basic elements can be arranged in the following way:
![image](https://github.com/jintonic/shine/assets/4230658/e6ac187d-ebab-4f90-a7ed-069c0b8e1883)

**Comment by bestdeveloper23** [comment]
https://monosnap.com/file/r7zqQcq3RxZ31EPd7kleGkKIvWtm3U

https://monosnap.com/file/posIIBUmCGJY86NZsZcG8gzOM88shj
We are using up to  98td element.
Is it right?
So I have to make two options for basic elements and compounds?

**Comment by jintonic** [comment]
Yes, up to 98. Please work on the periodic table first and keep the compound list as it is. I'll think about a better way for compound materials.

**Comment by bestdeveloper23** [comment]
I see.

**Comment by bestdeveloper23** [comment]
https://monosnap.com/file/ym95CcYbV3sUUSe4DmLHP8A0JTRg0x
Should I add atomic weight too?

**Comment by jintonic** [comment]
No need. You can keep the density and energy information that you added before there.

**Comment by bestdeveloper23** [comment]
https://monosnap.com/file/MGTTMCJ5RvQ2YgoKWfBNCUdyDpgHai
Which design do you have in mind?
I think it would be better to make it like popup.

**Comment by bestdeveloper23** [comment]
I completed basic elements arrangement for 19 hours.

**Comment by bestdeveloper23** [comment]
https://monosnap.com/file/xKqXYEAICsHQ0niwDlNW2x7AGpcTRf
You can check this.

**Comment by jintonic** [comment]
It looks very nice! But can we change it to something like this?
![image](https://github.com/jintonic/shine/assets/4230658/4172441b-1741-42a6-860c-5df624b7c012)

**Comment by bestdeveloper23** [comment]
https://monosnap.com/file/zV5pFluSIyto8VxFAVLAq4eYNNTVuO
Like this?

**Comment by jintonic** [comment]
34 -19 -1 = 14 hours left. Right?

**Comment by bestdeveloper23** [comment]
Yes.
I also worked #30 for 9 hours.

**Comment by jintonic** [comment]
14 - 9 = 5 hours left
